packages/zwave-js/src/lib/commandclass/SceneControllerConfigurationCC.ts

```diff
@@ -148,9 +148,27 @@ export class SceneControllerConfigurationCCAPI extends CCAPI {
 					) ?? new Duration(0, "seconds");
 				await this.set(propertyKey, value, dimmingDuration);
 			}
+		} else if (property === "dimmingDuration") {
+			if (typeof value !== "number") {
+				throwWrongValueType(
+					this.ccId,
+					property,
+					"duration",
+					typeof value,
+				);
+			}
+
+			const dimmingDuration = Duration.parseSet(value);
+
+			const node = this.endpoint.getNodeUnsafe()!;
+			// If SceneId missing, we need to set a default to 0,
+			// which disables the scene configuration for the association group
+			const sceneId =
+				node.getValue<number>(
+					getSceneIdValueID(this.endpoint.index, propertyKey),
+				) ?? 0;
+			await this.set(propertyKey, sceneId, dimmingDuration);
```

Comment:
I'm not sure if we should send a command here if the scene ID isn't set yet. If a user sets `dimmingDuration`, followed by `scene ID`, I think we could just save the duration value ID locally until it is used in the next command.

What do you think?

Reply:
Looking closer at the specs, `dimmingDuration` SHOULD be `0` if `sceneId` is `0`, so my code here breaks the recommendations.

The two valid options are your suggestion of saving the value locally or throwing an error. I think throwing an error would surprise users who arbitrarily set the values in the wrong order, so I'll store it in the `valueDB` where it can get picked up by a subsequent `SET_VALUE` call that sets the `sceneId` property.